ods_ci/tests/Tests/0600__distributed_workloads/test-run-distributed-workloads-tests.robot

```diff
@@ -92,3 +92,13 @@ Run TestKueueCustomRayRocmGpu ODH test with Python 3.11
     ...     Training
     ...     WorkloadsOrchestration
     Run DistributedWorkloads ODH Test    TestMnistCustomRayRocmGpu    ${RAY_TORCH_ROCM_IMAGE_3.11}    ${NOTEBOOK_IMAGE_3.11}
+
+Run TestValidatingAdmissionPolicy ODH test
+    [Documentation]    Run Go ODH test: TestValidatingAdmissionPolicy
+    [Tags]  Tier2
+    ...     DistributedWorkloads
+    ...     WorkloadsOrchestration
+    ...     MYTEST
```

Comment:
Should be removed?

Reply:
@sutaakar 
Thank you for noticing it! Tag was removed.